3DAmsterdam/Assets/Amsterdam3D/Scripts/OBJLoader/ObjStringLoader.cs

```diff
@@ -30,64 +37,113 @@ private void Start()
 #if UNITY_EDITOR
 		private void Update()
 		{
+			//Only used for in editor testing loading of local model files
```

Comment:
Het is test code, maar misschien is het handig om de file paths in iets van Application.datapath te stoppen? Dan kunnen we test files in de repo zetten zodat wij deze functies ook kunnen testen etc

Reply:
Goeie! Ga ik doen